frontend/src/pages/Blog.tsx

```diff
@@ -0,0 +1,33 @@
+import Navigation from "../components/Navigation";
+
+
+export default function Blog() {
+
+    const handleSubmit = () => {
+
```

Comment:
- what does `handleSubmit` do?
- isn't there a need for some variables to keep track of what the user inputs and some eventHandlers like `onChange`?